tests/jballerina-integration-test/src/test/resources/observability/metrics_tests/commons.bal

```diff
@@ -14,11 +14,13 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/observe.mockextension;
 import ballerina/testobserve;
 
 service /metricsRegistry on new testobserve:Listener(10090) {
     resource function post getMetrics(testobserve:Caller caller) {
-        json metrics = testobserve:getMetrics();
-        checkpanic caller->respond(metrics.toJsonString());
+        mockextension:Metrics metrics = mockextension:getMetrics();
+        json metricsJson = checkpanic metrics.cloneWithType(json);
+        checkpanic caller->respond(metricsJson.toJsonString());
```

Comment:
Can't we directly respond with json type ?

Reply:
Now we get the Metrics record instead of the json from the mockextension, we have to convert it to json before responding right?

Reply:
Yeah... We can directly respond with the json IINM.
```
checkpanic caller->respond(metricsJson);
```